src/components/CreateBook/Createbookform.jsx

```diff
@@ -1,14 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Createbookform.css';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../../redux/books/booksSlice';
 
 function Createbookform() {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const [category, setCategory] = useState('');
+  const dispatch = useDispatch();
+
+  const formData = {
+    item_id: uuidv4(),
+    title,
+    author,
+    category,
+  };
+
+  const handleSubmit = () => {
+    // e.preventDefault();
```

Comment:
Great job @Peter-Dumbari 🤝

- [ ] Kindly remove this commented line of code as this is regarded as a bad practice.